components/evaluate_ragas/fondant_component.yaml

```diff
@@ -19,13 +19,15 @@ produces:
 
 
 args:
-  module:
+  llm_module_name:
     description: Module from which the LLM is imported. Defaults to langchain.llms
     type: str
-    default: "langchain.llms"
-  llm_name:
+    default: "langchain.chat_models"
```

Comment:
By doing this we are deciding to use gpt-4 as default. Do you know why we have used gpt-3.5 as default before?

Reply:
gpt-4 is more expensive. I would use 3.5 by default if possible.

Reply:
changed to 3.5, yes but I only tested the new module with 3.5/4 on the old component. It should be fine since the only thing that's changed is the naming in this PR and the defaults. Will include the new parameters when we release